db/schema.rb

```diff
@@ -98,8 +97,14 @@
     t.text "visit_dates"
     t.text "referral_source"
     t.bigint "location_id", null: false
+    t.string "parent_type"
```

Comment:
This stuff from the old PR using polymorphic association should not be here. I think this is because you are using the same database from your old PR and it is including this in the schema. I suggest dropping and recreating the table so it builds only with the migrations you want it to for this PR.

Reply:
I had to reset it because of making commits to the wrong branch. Monday - I think this cleaner now